doc/man/markdown/cockpit-desktop-a.1

```diff
@@ -0,0 +1,77 @@
+'\" t
+.\"     Title: cockpit-desktop
+.\"    Author: [see the "AUTHOR(S)" section]
+.\" Generator: Asciidoctor 2.0.20
+.\"      Date: 2024-10-28
+.\"    Manual: \ \&
+.\"    Source: \ \&
+.\"  Language: English
+.\"
+.TH "COCKPIT\-DESKTOP" "1" "2024-10-28" "\ \&" "\ \&"
+.ie \n(.g .ds Aq \(aq
+.el       .ds Aq '
+.ss \n[.ss] 0
+.nh
+.ad l
+.de URL
+\fI\\$2\fP <\\$1>\\$3
+..
+.als MTO URL
+.if \n[.g] \{\
+.  mso www.tmac
+.  am URL
+.    ad l
+.  .
+.  am MTO
+.    ad l
+.  .
+.  LINKSTYLE blue R < >
+.\}
+.SH "NAME"
+cockpit-desktop \- Cockpit Desktop integration
+.SH "SYNOPSIS"
+.sp
+\fBcockpit\-desktop\fP {URLPATH} [SSH_HOST]
+.SH "DESCRIPTION"
+.sp
+The \f(CRcockpit\-desktop\fP program provides secure access to Cockpit pages in
+an already running desktop session. It starts a web server
+(\f(CRcockpit\-ws\fP) and a web browser in an isolated network namespace, and a
+\f(CRcockpit\-bridge(8)\fP in the running user session.
+.sp
+This avoids having to log into Cockpit, and having to enable
+\f(CRcockpit.socket\fP system\-wide. The network isolation ensures that no
+other user, and not even other processes in the user\(cqs session, can
+access this local web server.
+.sp
+\f(CRURLPATH\fP is the Cockpit page to open, i. e. the path component of
+Cockpit URLs. It is highly recommended to only open a
+.URL "https://cockpit\-project.org/guide/latest/embedding.html" "particular page frame" ","
+not the entire Cockpit navigation and menu. For example, the
+path \f(CR/cockpit/@localhost/storage/index.html\fP will open the Storage
+page. It is also possible to give abbreviated forms of urls, such as
+\(lqstorage\(rq or \(lqnetwork/firewall\(rq.
+.sp
+\f(CRSSH_HOST\fP is an optional SSH remote host specification (\f(CRhostname\fP or
+\f(CRusername@hostname\fP). If given, \f(CRcockpit\-bridge\fP will be started on the
+remote host through \f(CRssh(1)\fP instead, i. e. the Cockpit web browser will
+show that remote host. Note that this is more of an experimental/demo
+feature.
+.SH "ENVIRONMENT"
+.sp
+The \f(CRBROWSER\fP environment variable specifies the browser command (and
+possibly options) that will be used to open the requested Cockpit page.
+If not set, \f(CRcockpit\-desktop\fP attempts to use an internal minimalistic
+WebKit browser, and failing that, will attempt to detect some reasonable
+alternatives.
+.SH "BUGS"
+.sp
+Please send bug reports to either the distribution bug tracker or the
+.URL "https://github.com/cockpit\-project/cockpit/issues/new" "upstream bug tracker" "."
```

Comment:
This is the raw output of asciidoctor.

It renders URL correctly in the man markup using `.URL` keyword.